index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Website</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+  </head>
+  <body>
+
+  <!-- HEADER SECTION (major section) -->
+
+  <header id="headerSection">
+
+  <!-- navigation area (subsection) -->
+    <nav id="navBar">
+      <div id="logoContainer" class="navitems">
+        <p id="logo">My Logo</p>
+      </div>
+
+      <div id="menuiconContainer" class="navItems">
+      	<img src="./media-library/Union.png" alt="menu" width="" height="">
+      </div>
+
+    </nav>
+
+    <!-- about area (subsection) -->
+    <section id="aboutContainer">
+      <div id="aboutTitle">
+        <p>I’m Alexandra. Glad to see you!</p> 
```

Comment:
- Kindly the main title (e.g., "I'm Marianne ...") is an h1 tag. instead of a p tag , p tag is only for paragraph :+1: 

If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.